packages/db/tests/lib/test-file.js

```diff
@@ -11,8 +11,12 @@ const {
   fakeGranuleRecordFactory,
   getFilesAndGranuleInfoQuery,
   migrationDir,
+  fakeProviderRecordFactory,
+  ProviderPgModel,
 } = require('../../dist');
 
+const randomBucket = () => `bucket-${cryptoRandomString({ length: 10 })}`;
```

Comment:
Nit: *slight/mild* preference for randomBucketName

Reply:
8b7ca1a2af